src/main/resources/static/_shared.yml

```diff
@@ -2904,7 +2904,7 @@ components:
         isWithdrawn:
           type: boolean
         domainEvent:
-          type: object
+          $ref: '#/components/schemas/AnyValue'
```

Comment:
Could we maybe structure this a bit better? 

- id
- eventType = string
- timestamp = datetime
- updatedFields = array of objects with fieldName, updatedTo, updatedFrom